simulation/executor/internal/executortypes/simmanager.go

```diff
@@ -165,7 +169,8 @@ func (m Manager) GenerateGenesisStates(simState *module.SimulationState, sim *si
 			if mod, ok := simModule.(simtypes.AppModuleSimulationGenesis); ok {
 				mod.SimulatorGenesisState(simState, sim)
 			} else {
-				simState.GenState[simModule.Name()] = simModule.DefaultGenesis(simState.Cdc)
+				// UNFORKINGTODO OQ: Figure out how to call DefaultGenesis, I don't understand
+				// simState.GenState[simModule.Name()] = simModule.DefaultGenesis(simState.Cdc)
```

Comment:
hrmm this idk either, but I think we can punt fixing the simulator in general

Reply:
Changed to unforkingnote https://github.com/osmosis-labs/osmosis/pull/6413/commits/2e1c779ac8cb2bf84cc0b1f1c9d0faaffc7d30aa